.github/workflows/build.yml

```
@@ -329,7 +329,7 @@ jobs:
         run: cd dist && sha256sum * > "k6-${VERSION}-checksums.txt"
       - name: Anchore SBOM Action
         continue-on-error: true
-        uses: anchore/sbom-action@v0.12.0
+        uses: anchore/sbom-action@v0.15.9
```

non-blocking, maybe use hash (third-party action)? :thinking: 

```suggestion
        uses: anchore/sbom-action@9fece9e20048ca9590af301449208b2b8861333b # v0.15.9
```